test/tests/performance/pbs_jobperf.py

```diff
@@ -0,0 +1,589 @@
+            thrds = []
+            for u in range(0, config['No_of_users']):
+                t = Thread(target=self.submit_jobs,
+                           args=(users[u], config['No_of_jobs_per_user']))
+                t.start()
+                thrds.append(t)
+            for t in thrds:
+                t.join()
+
+            start = time.time()
+            bin_path = str(os.path.join(
+                self.server.pbs_conf['PBS_EXEC'], 'bin'))
+            qdel = str(os.path.join(bin_path, 'qdel'))
+            if config['qdel_exec_args']:
+                qdel = qdel + ' -W force'
+            cmd = qdel + " `" + str(os.path.join(bin_path, 'qselect')) + "`"
+            subprocess.call(cmd, shell=True)
+            stop = time.time()
+            res = stop - start
+            qdel_time.append(res)
+            j = j + 1
+            avg_qdel_time.extend(qdel_time)
+        self.perf_test_result(avg_qdel_time, "job_deletion", "secs")
+
+    @timeout(3600)
+    def test_qdel_hist_perf(self):
+        """
+        Test job deletion performance for history jobs
+        """
+        config = {}
+        testconfig = {'No_of_jobs_per_user': 100,
+                      'No_of_tries': 1,
+                      'No_of_users': 10,
+                      'svr_log_level': 511,
+                      'No_of_moms': 21,
+                      'No_of_ncpus_per_node': 48}
+        config = self.set_test_config(testconfig)
+
+        avg_qdel_time = []
+        qdel_time = []
+        j = 0
+        nodes = self.server.status(NODE, 'state=free')
+        if len(nodes) < config['No_of_moms']:
+            a = {'resources_available.ncpus': config['No_of_ncpus_per_node']}
+            self.server.create_moms('mom', a, config['No_of_moms'], self.mom)
+        while j < config['No_of_tries']:
+            users = ['pbsuser1', 'pbsuser2', 'pbsuser3', 'pbsuser4',
+                     'pbsuser5', 'pbsuser6', 'pbsuser7', 'pbstest',
+                     'pbsroot', 'pbsadmin']
+            a = {'log_events': config['svr_log_level']}
+            self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
+            a = {'scheduling': 'False'}
+            self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
+            thrds = []
+            for u in range(0, config['No_of_users']):
+                t = Thread(target=self.submit_jobs,
+                           args=(users[u], config['No_of_jobs_per_user']))
+                t.start()
+                thrds.append(t)
+            for t in thrds:
+                t.join()
+            log_start = int(time.time())
+            a = {'job_history_enable': 'True'}
+            self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
+            a = {'scheduling': 'True'}
+            self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
+            sclg = PBSLogAnalyzer()
+            md = sclg.analyze_scheduler_log(
+                filename=self.scheduler.logfile, start=int(log_start))
```

Comment:
log_start variable is already stored as _int_ initially. Is there a need to convert again explicitly?